image/templates/helm/stackrox-central/templates/01-central-08-db-configmap.yaml

```diff
@@ -0,0 +1,939 @@
+{{- include "srox.init" . -}}
+
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: central-db-config
+  namespace: {{ .Release.Namespace }}
+  labels:
+    {{- include "srox.labels" (list . "configmap" "central-db-config") | nindent 4 }}
+  annotations:
+    {{- include "srox.annotations" (list . "configmap" "central-db-config") | nindent 4 }}
+data:
+  postgresql.conf: |
+    {{- if ._rox.central.enableCentralDB }}
+    # -----------------------------
```

Comment:
Rather than putting the entire config inline in a YAML file, put the files in `config/central-db/postgresql.conf` and `config/central-db/pg_hba.conf`, and include them here via, e.g.,
```
data:
  postgresql.conf: |
    {{- .Files.Get "config/central-db/postgresql.conf" | nindent 4 }}
```

Reply:
will do

Reply:
Does this preclude that postgresql.conf from being templated?